CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
     * Add `VenmoClient#createPaymentAuthRequest`
     * Move `showVenmoInGooglePlayStore` to `VenmoLauncher`
     * Remove `isVenmoAppSwitchAvailable` and `isReadyToPay` (no longer required as Venmo flow will fallback to web if app is not installed)
+    * Update package name to `com.braintreepayments.api.venmo`
```

Comment:
should this go to the top of the list?

Reply:
I've been adding all of the package name updates to the bottom of each section. Should we surface them to the top?

Reply:
I moved all of the package rename bullets to the top of each section 👍